hdr_html_canvas_element.md

```diff
@@ -18,28 +18,63 @@ There are four main classes of HDR use that inform this proposal. They are:
 
 There exist the following constraints.
 
-* The exact maximum luminance of the output display is not known and not knowable.
+* The exact maximum luminance of the output display is not knowable without user permission.
   * The [CSS Media Queries Level 5 Specification](https://www.w3.org/TR/mediaqueries-5/#valdef-media-dynamic-range-high) allows the application to query the ``'dynamic-range'``. The resulting values are ``'standard'`` and ``'high'``.
   * The value changes over time.
   * The exact value is a fingerprinting vector.
-* The exact number of nits of SDR content is also not known and not knowable.
+* The exact number of nits of SDR content is also not knowable without user permission.
   * On macOS, it is always 100.
   * On Windows, it depends on a user slider setting.
   * The exact value is a fingerprinting vector (again).
 
-Because these values are not known, it is not possible for the application provide quantities related to display light.
-
 ### Proposed solution overview
 
 The solution that we propose is to:
 
+* Add attributes to the `ScreenAdvanced` interface to expose screen parameters relevant to high dynamic range and wide color gamut.
 * Introduce new color spaces and precisions that are useful for HDR.
 * Clearly define invertible and context-independent transformations between these spaces.
 * Introduce the ability use more than 8 bits per pixel for a canvas element.
 * Introduce the ability for an `HTMLCanvasElement` to configure HDR.
 
 ## Proposal
 
+### Querying screen high dynamic range parameters
+
+Add a new attribute to `ScreenAdvanced` to indicate the HDR headroom currently available on the screen.
+
+```idl
+  partial interface ScreenAdvanced {
+    // The maximum luminance that the screen is capable of displaying, as a
```

Comment:
Same comment as above.

Reply:
Clarified that this is just fullscreen (only one platform exposes other values)